tests/src/Traits/ClientTestTrait.php

```diff
@@ -20,13 +23,13 @@
 trait ClientTestTrait
 {
     /**
-     * @var array
+     * @var array<int, array{request: Request, response: Response, error: mixed, options: mixed[]}>
```

Comment:
I added the long list when I was looking PHPStan level 9. Actually you are right, it is too verbose for level 8, and the usage of this array is strictly typed, so we get all the type hinting in PHPStorm.

I'll change it to `@var array<int, array<string, mixed>>`, to avoid PHPStorm throwing warnings that `array<mixed> is equivalent to array`,